src/renderer/pages/player-page.js

```diff
@@ -722,6 +726,18 @@ function renderPlayerControls (state) {
     }
   }
 
+  // Handles a scrub hover (preview another position in the video)
+  function handleScrubPreview (e) {
+    // Only show for videos
+    if (!e.clientX || state.playing.type !== 'video') return
+    dispatch('mediaMouseMoved')
+    dispatch('preview', e.clientX)
+  }
+
+  function clearPreview (e) {
+    dispatch('clearPreview')
```

Comment:
I would add the same check that in `handleScrubPreview` 😉
```
if (state.playing.type !== 'video') return
```

Reply:
Done. Thank you 🙏